docs/developers/bridges/usdcbridge.md

```diff
@@ -0,0 +1,169 @@
+---
+title: USDC Bridge
+sidebar_position: 2
+---
+
+# Introduction
+
+### The Circle USDC Bridge on Linea
+
+This article is about a crypto-asset bridge that has been deployed to Linea. It has been designed and built by [ConsenSys](https://consensys.net/), to the specifications of [Circle](https://www.circle.com/en/), for the purposes of bridging [USDC](https://www.circle.com/en/usdc) between networks: initially, between the [Goerli Ethereum Testnet](https://goerli.net/) and the [Linea Goerli Testnet](https://linea.build/).
```

Comment:
"It has been designed and built by ConsenSys to the specifications of Circle"
-> I would rather say it has been built in partnership with Circle.
We should validate this doc with Circle before publishing it (slack #ext-zkevm-circle-bridge)

Reply:
OK, will do!